core/types/gen_header_json.go

```diff
@@ -72,25 +77,27 @@ func (h Header) MarshalJSON() ([]byte, error) {
 // UnmarshalJSON unmarshals from JSON.
 func (h *Header) UnmarshalJSON(input []byte) error {
 	var dec struct {
-		ParentHash    []common.Hash    `json:"parentHash"          gencodec:"required"`
-		UncleHash     []common.Hash    `json:"sha3Uncles"          gencodec:"required"`
-		Coinbase      []common.Address `json:"miner"               gencodec:"required"`
-		Root          []common.Hash    `json:"stateRoot"           gencodec:"required"`
-		TxHash        []common.Hash    `json:"transactionsRoot"    gencodec:"required"`
-		ReceiptHash   []common.Hash    `json:"receiptsRoot"        gencodec:"required"`
-		EtxHash       []common.Hash    `json:"extTransactionsRoot" gencodec:"required"`
-		EtxRollupHash []common.Hash    `json:"extRollupRoot"       gencodec:"required"`
-		ManifestHash  []common.Hash    `json:"manifestHash"        gencodec:"required"`
-		Bloom         []Bloom          `json:"logsBloom"           gencodec:"required"`
-		Difficulty    []*hexutil.Big    `json:"difficulty"         gencodec:"required"`
-		Number        []*hexutil.Big    `json:"number"             gencodec:"required"`
-		GasLimit      []hexutil.Uint64 `json:"gasLimit"            gencodec:"required"`
-		GasUsed       []hexutil.Uint64 `json:"gasUsed"             gencodec:"required"`
-		BaseFee       []*hexutil.Big    `json:"baseFeePerGas"      gencodec:"required"`
-		Location      common.Location   `json:"location"           gencodec:"required"`
-		Time          hexutil.Uint64   `json:"timestamp"           gencodec:"required"`
-		Extra         hexutil.Bytes    `json:"extraData"           gencodec:"required"`
-		Nonce         BlockNonce       `json:"nonce"`
+		ParentHash    		[]common.Hash    	`json:"parentHash"          gencodec:"required"`
+		UncleHash     		[]common.Hash    	`json:"sha3Uncles"          gencodec:"required"`
+		Coinbase      		[]common.Address 	`json:"miner"               gencodec:"required"`
+		Root          		[]common.Hash    	`json:"stateRoot"           gencodec:"required"`
+		TxHash        		[]common.Hash    	`json:"transactionsRoot"    gencodec:"required"`
+		ReceiptHash   		[]common.Hash    	`json:"receiptsRoot"        gencodec:"required"`
+		EtxHash       		[]common.Hash    	`json:"extTransactionsRoot" gencodec:"required"`
+		EtxRollupHash 		[]common.Hash    	`json:"extRollupRoot"       gencodec:"required"`
+		ManifestHash  		[]common.Hash    	`json:"manifestHash"        gencodec:"required"`
+		Bloom         		[]Bloom          	`json:"logsBloom"           gencodec:"required"`
+		Difficulty    		*hexutil.Big    	`json:"difficulty"         	gencodec:"required"`
+		ParentEntropy    	[]*hexutil.Big 		`json:"parentEntropy"	   	gencodec:"required"`
```

Comment:
gofmt -w -s .

Reply:
thanks, will do, its formatted in my laptop, and not on github, its confusing